src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
This is worth discussing. If a node in a cluster runs out of disk space, we currently alarm all nodes. The reasoning behind this is fairly easy to guess: any client can connect to any node and publish in a way that would route messages to the alarmed node.

However, I can see how we may want to be more permissive in this particular case. Unlike running out of RAM, where
any incoming message can put us over the limit, with disk it's less likely to be the case.